### ISSUE TITLE
Chapter 14

### ISSUE THREAD

**Issue by zhwbin65** [matheusfacure/python-causality-handbook#202]
There is an issue on chapter 14, in the following paragraph
[paragraph here]
Another less obvious case when fixed effect fails is when you have reversed causality. For instance, let’s say that it isn’t marriage that causes you to earn more. Is earning more that increases your chances of getting married. In this case, it will appear that they have a positive correlation but earnings come first. They would change in time and in the same direction, so fixed effects wouldn’t be able to control for that.

It should be
[suggestion here]
This statement is kind of misieading. As long as the analytics steps follows FE frame rigously, The revered causality would not happen. In the sample, it happens when the time of marriage status change and earnings change can not be specified, that is we cannot know which happen first. 
Reversed causality is not only specific for FE but for all causal analysis. When we do causal analysis, we expect change only happens after treatment.

**Comment by matheusfacure** [comment]
Fair point. This issue will take more time to be fixed. I'm writing a section on panel data that will address it more thoroughly.
